cond_notin.go

```diff
@@ -20,10 +20,8 @@ func NotIn(col string, values ...interface{}) Cond {
 }
 
 func (condNotIn condNotIn) handleBlank(w Writer) error {
-	if _, err := fmt.Fprintf(w, "%s NOT IN ()", condNotIn.col); err != nil {
-		return err
-	}
-	return nil
+	_, err := fmt.Fprint(w, "0=0")
```

Comment:
Wouldn't that delete all users (i.e. every user whose id is not in the empty list)? So we would want `"0=0"`, not `"1=0"`.